ConsoleApp1/VehicleParts/Vehicle.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp1.VehicleParts
+{
+    internal class Vehicle
+    {
+        public enum VehicleType
+        {
+            Auto,
+            Motorcycle
+        }
+        
+        public enum VehicleStatus
+        {
+            Arrived,
+            InRepairing,
+            GotRepaired,
+            Issued
+        }
+
+        public string Id { get; set; }
+
+        public VehicleType Type { get; set; }
+
+        public VehicleStatus Status { get; set; }
+
+        public Engine Engine { get; set; }
+        public List<Wheel> Wheels { get; set; }
+
+        public void ChangeStatus(VehicleStatus vehicleStatus)
+        {
+            throw new NotImplementedException();
+        }
+
+        public void IncludeMainWorker(Worker worker)
```

Comment:
Regarding to your scheme Vehicle should aggregate Worker, but here is association

Reply:
So either adjust scheme, either code

Reply:
yep. youre right.